homeassistant/components/cover/myq.py

```diff
@@ -21,11 +21,17 @@
 
 DEFAULT_NAME = 'myq'
 
+STATE_STOPPED = 'stopped'
+STATE_TRANSITION = 'transition'
+
 MYQ_TO_HASS = {
     'closed': STATE_CLOSED,
-    'open': STATE_OPEN,
     'closing': STATE_CLOSING,
-    'opening': STATE_OPENING
+    'open': STATE_OPEN,
+    'opening': STATE_OPENING,
+    'stopped': STATE_STOPPED,
+    'transition': STATE_TRANSITION,
+    'unknown': STATE_UNKNOWN
```

Comment:
Don't use `STATE_UNKNOWN`. Use `None` to represent unknown state. So remove this line and use `dict.get` in the look up instead.

Reply:
Makes sense, thanks for the help.